tests/BenchmarkDotNet.Tests/ConfigParserTests.cs

```diff
@@ -271,10 +271,10 @@ public void IlCompilerPathParsedCorrectly()
         }
 
         [Theory]
-        [InlineData("netcoreapp3.1", true)]
-        [InlineData("net5.0", true)]
-        [InlineData("net6.0", true)]
-        [InlineData("net8.0", true)]
+        [InlineData("netcoreapp2.0", true)]
+        [InlineData("netcoreapp2.1", true)]
+        [InlineData("netcoreapp2.2", true)]
+        [InlineData("netcoreapp3.0", true)]
```

Comment:
instead of replacing the tfms, we should test all of them (just add new test cases)